beast/plotting/plot_noisemodel.py

```diff
@@ -151,7 +151,7 @@ def plot_noisemodel(
         help="path+name of the noise model file(s)",
     )
     parser.add_argument(
-        "phot_file", type=str, help="name of the file to save the plot",
+        "plot_file", type=str, help="name of the file to save the plot",
```

Comment:
It looks like this is supposed to be `phot_file`.  Was this accidentally changed?  Or maybe the `args.phot_file` below needs to be updated too.

Reply:
Nope, supposed to be *plot* as it is the file to save the plot to.  Having it as phot confused me.  Will fix the args variable name.  Good catch.